consensus/core/src/block_manager.rs

```diff
@@ -988,6 +988,10 @@ mod tests {
             Ok(())
         }
 
+        fn check_transactions(&self, _transactions: &[&[u8]]) -> ConsensusResult<()> {
```

Comment:
I prefer not to have an interface func just for testing. Is it possible to verify the returned error instead? Or make this a non-pub func on `SignedBlockVerifier` instead. I have been considering calling transaction verifier out of block verifier. It will be weird to split transaction verification to multiple callsites. Maybe they need to be recombined later.